extensions/EXT_texture_compression_bptc/extension.xml

```diff
@@ -64,7 +64,7 @@
           <dt>COMPRESSED_RGB_BPTC_SIGNED_FLOAT_EXT</dt>
           <dt>COMPRESSED_RGB_BPTC_UNSIGNED_FLOAT_EXT</dt>
           <dd>
-          <p>When <code>level</code> equals zero <code>width</code> and <code>height</code>
+          <p><code>width &lt;&lt; level</code> and <code>height &lt;&lt; level</code>
           must be a multiple of 4.</p>
```

Comment:
Here and throughout: I'm not sure this encodes the constraints discussed in #3117. Rereading the discussion there, it sounds like two branches are needed.
 - For both WebGL 1.0 and 2.0 contexts, the base level's width and height must each be a multiple of 4.
 - For WebGL 1.0 contexts, the base level's width and height must each be a power of two.

This should apply to all of these texture formats. DXT in particular needs to retain the power-of-two restriction for WebGL 1.0 contexts. WebGL's BPTC and RGTC extensions are written against the 1.0 spec even though WebGL 1.0 doesn't have RG textures, so theoretically they should retain the same restrictions.

Reply:
I'm not really convinced by the webgl1 pot requirement, which I don't think in enforced or tested, and isn't spec'd. Patches welcome but I don't want to tackle that here.

Reply:
OK, I see, even though we discussed this in the earlier issue it wasn't enforced. These changes are fine then.